zds/tutorialv2/views/views_contents.py

```diff
@@ -68,8 +68,9 @@ def get_redirect_url(self, **kwargs):
 
 class CreateContent(LoggedWithReadWriteHability, FormWithPreview):
     """
-    Handle content creation. Since v22, we explicitely ask for user to choose a licence instead of \
-    assuming he just wants the default "All rights reserved" option.
+    Handle content creation. Since v22, we explicitely ask for user to choose a licence instead of
+    assuming he just wants the default "All rights reserved" option. Even though, the user can configure
+    a default licence in his profile.
```

Comment:
Il y a plusieurs fautes dans cette phrase.
Suggestion de @vhf : "Handle content creation. Since v22 a licence must be explicitly selected instead of defaulting to "All rights reserved". Users can however set a default licence in their profile."